modules/las/src/index.ts

```diff
@@ -13,10 +13,9 @@ export {LASWorkerLoader};
  */
 export const LASLoader = {
   ...LASWorkerLoader,
-  parse: async (arrayBuffer: ArrayBuffer, options?: LASLoaderOptions) =>
+  parse: async (arrayBuffer: ArrayBuffer, options: LASLoaderOptions) =>
     parseLAS(arrayBuffer, options),
-  parseSync: (arrayBuffer: ArrayBuffer, options?: LASLoaderOptions) =>
-    parseLAS(arrayBuffer, options)
+  parseSync: (arrayBuffer: ArrayBuffer, options: LASLoaderOptions) => parseLAS(arrayBuffer, options)
 };
-
+// @ts-ignore
```

Comment:
If you need to ignore this you have a problem in your loader definition. Some missing field presumably.